cling/cling_build_instructions.md

```diff
@@ -23,28 +23,21 @@ Alternatively, you can build manually.
 Start by checking out llvm, clang and cling:
 
 {% highlight C++ %}
-git clone http://root.cern/git/llvm.git src
-cd src
-git checkout cling-patches
-cd tools
-git clone http://root.cern/git/cling.git
-git clone http://root.cern/git/clang.git
-cd clang
-git checkout cling-patches
-cd ../..
+git clone -b cling-llvm13 https://github.com/root-project/llvm-project.git src
+git clone https://github.com/root-project/cling.git
 {% endhighlight %}
 
 Then use CMake to configure &amp; build cling:
 
-{% highlight C++ %}
+{% highlight shell %}
 mkdir build
 cd build
-cmake -DCMAKE_INSTALL_PREFIX=[Install Path] -DCMAKE_BUILD_TYPE=[Build configuration, e.g. Release or Debug] ..\src
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DLLVM_BUILD_TOOLS=Off -DLLVM_EXTERNAL_PROJECTS=cling -DLLVM_EXTERNAL_CLING_SOURCE_DIR=../cling -DLLVM_ENABLE_PROJECTS=clang -DLLVM_TARGETS_TO_BUILD="host;NVPTX" src/llvm
 cmake --build .
 cmake --build . --target install
 {% endhighlight %}
 
-### cling-patches?
+### cling patches?
 
 Cling needs a couple of patches on top of clang (but can work with an original build of llvm).
```

Comment:
This is wrong for LLVM 13, we need patches in LLVM and an original build will not work.